src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/ModelMetadataValidationDetails.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding
+{
+    public class ModelMetadataValidationDetails
+    {
+        public bool IsRequired { get; set; }
```

Comment:
Good catch. It's not. This class is going to be empty for now. That's my next project (fill it up).